packages/next/cli/next-info.ts

```diff
@@ -52,9 +55,26 @@ const nextInfo: cliCommand = async (argv) => {
       Yarn: ${getBinaryVersion('yarn')}
       pnpm: ${getBinaryVersion('pnpm')}
     Relevant packages:
-      next: ${getPackageVersion('next')}
+      next: ${installedRelease}
       react: ${getPackageVersion('react')}
-      react-dom: ${getPackageVersion('react-dom')}`)
+      react-dom: ${getPackageVersion('react-dom')}
+`)
+
+  const res = await fetch(
+    'https://api.github.com/repos/vercel/next.js/releases'
+  )
+  const releases = await res.json()
+  const newestRelease = releases[0].tag_name.replace(/^v/, '')
+
+  if (installedRelease !== newestRelease) {
+    console.warn(
+      `${chalk.yellow(
+        chalk.bold('warn')
+      )}  - Next.js is outdated, detected: "${installedRelease}", newest: "${newestRelease}.
+        Please install the latest canary version (\`npm install next@canary\`) to confirm the issue still exists before creating a new issue.
+        Read more - https://nextjs.org/docs/messages/opening-an-issue`
```

Comment:
```suggestion
      )}  - Latest canary version not detected, detected: "${installedRelease}", newest: "${newestRelease}.
        Please install the latest canary version (\`npm install next@canary\`) to confirm the issue still exists before creating a new issue.
        Read more - https://nextjs.org/docs/messages/opening-an-issue`
```
This sounds like we are encouraging canary use in general, might be good to tweak this a bit.

Reply:
Sounds good. how about also adding "Please try the latest canary version..."